.gitlab-ci.yml

```diff
@@ -893,7 +898,15 @@ test_backend_integration:
     reports:
       junit: results_backend_integration_*.xml
 
-test_full_integration:
+test_backend_integration_open:
```

Comment:
Naming:
I think `test_backend_integration:open_source` is nicer, and spending a few extra letters here is not really a cost to us :)